components/CDPUserDashboard.tsx

```diff
@@ -109,36 +153,45 @@ const CDPUserDashboard = (props: Props): JSX.Element => {
     // Negative sorts "reverse alphabetically" which allows newest files to appear first
     .sort((a: CDPFile, b: CDPFile) => -a.key.localeCompare(b.key))
 
+  const isSurveyResponse = (wanted: boolean) => (file: CDPFile) =>
```

Comment:
Add a comment: what does `wanted` do?

Reply:
Good catch thank you I had no idea so have now added a comment